plugins/modules/swim_workflow_manager.py

```diff
@@ -1690,13 +1920,20 @@ def verify_diff_imported(self, import_type):
         """
 
         if import_type == "remote":
-            image_name = self.want.get("url_import_details").get("payload")[0].get("source_url")
-        else:
+            image_names = self.want.get("url_import_details").get("payload")[0].get("source_url")
```

Comment:
image_names = self.want.get("url_import_details", {}).get("payload", [{}])[0].get("source_url", [])

Reply:
updated the code